backend/prisma/schema.prisma

```diff
@@ -7,31 +7,110 @@ generator prisma_client {
   provider = "prisma-client-js"
 }
 
-enum Status {
-  ADMIN MODERATOR FREE_USER PRO_USER
+model User {
+  Id                String        @id @default(cuid())
+  Name              String
+  Email             String        @unique
+  Password          String
+  ResetToken        String?
+  ResetTokenExpiry  Float?
+  userRole          UserRole      @default(FREE_USER)
+  CreatedAt         DateTime      @default(now())
+  UpdatedAt         DateTime      @updatedAt
+  Handle            String        @unique
+  Bio               String
+  SocialMedia       SocialMedia[]
+  interests         Topic[]
+  location          Location?
+  languagesNative   Language[]    @relation("LanguageToNative")
```

Comment:
The way I'm doing it is more thought-out than that, but still totally valid discussion 😊 The convention I'm using currently is that fields that are a scalar type are PascalCased and types that are custom types are camelCased, which gives you an indication just looking at the field name whether there's a relationship or not.

I wouldn't want to do `snake_case` here because this is going to end up as generated TypeScript/JavaScript code and we'll be potentially using those operations in our resolvers, etc.

Reply:
Definitely agree about `snake_case` - I thought maybe the fields would be converted automatically to `camelCase` for use in the frontend JS. But after seeing #28 and how it's used: ❌~snake_case~ ❌ 😆 

I think your idea of indicating scalar vs custom type with different `cases` is pretty clever 🦊 My only concerns are that it could be awkward using both cases when accessing values as properties, e.g. `post.Name` vs `user.languagesNative`, and conventionally, properties/fields are lowercase, with `PascalCase` implying that something is a Model or object instance. I also think it's actually a good thing to refer back to the schema file if there is ever a need for knowing if a field is scalar or custom. My vote would be to keep it simple and align with the style in the Prisma docs because it could be really hard to change later. Che ne pensi?

Reply:
Yep I'm fine with that! It was definitely more something I was trying out, but after using it for a while and seeing the experience on the client-side, too, I agree with your suggestion 😄